internal/gatewayapi/listener.go

```diff
@@ -144,3 +155,67 @@ func (t *Translator) ProcessListeners(gateways []*GatewayContext, xdsIR XdsIRMap
 		}
 	}
 }
+
+func processAccessLog(envoyproxy *configv1a1.EnvoyProxy) *ir.AccessLog {
+	if envoyproxy != nil && len(envoyproxy.Spec.AccessLoggings) > 0 {
+		irAccessLog := &ir.AccessLog{}
+		// translate the access log configuration to the IR
```

Comment:
no default case ?

Reply:
IMO, disabled accesslog by default.
The user should explicitly set accesslog configuration, especially for a Edge gateway.

Reply:
enabling access logs by default especially some format like `field:value`, adds a lot of value on top on Envoy. This does however make the API complex e.g how to disable default

Reply:
thoughts on this approach
* if accessLogs is null , we add a default text(or json) + file based Access log
* if Len(acesslogs) is 0 `accessLogs: []` , we dont add any defaults
* else if the user has set `accessLogs` in the API, we use that, and dont append any default
?

Reply:
+1, I think needs default case.

Reply:
for me, it's odd to have configuration like `[]` to disable somehting
```
apiVersion: config.gateway.envoyproxy.io/v1alpha1
kind: EnvoyProxy
metadata:
  name: disable-access-logging
spec:
  accessLoggings: []
```

Reply:
we discussed this in the community meeting and all agreed that having defaults is very useful.
But using `[]` is awkward and not an obvious way to disable `accessLogs`, one way to do this would be to create an explicit `disabled` field
```
accessLogs:
  disabled: true
  settings:
  - format: 
    sinks:
```